### PR TITLE
fix(zoro): Fix megacloud extractor

### PR DESCRIPTION
**Pull request by shimizudev**
**What kind of change does this PR introduce?**
Fixes https://github.com/consumet/consumet.ts/issues/623
Fixes https://github.com/consumet/consumet.ts/issues/624

**Did you add tests for your changes?**

**If relevant, did you update the documentation?**

**Summary**

**Other information**

### CODE REVIEW

File: bun.lockb
Comment (riimuru) [inline bun.lockb:1]: This file is not needed

File: package.json [hunk 1]
```diff
@@ -60,7 +60,8 @@
     "crypto-js": "^4.1.1",
     "domhandler": "^5.0.3",
     "form-data": "^4.0.0",
-    "husky": "^9.1.1"
+    "husky": "^9.1.1",
+    "image-pixels": "^2.2.2"
```
Comment (riimuru) [inline package.json:64]: It doesn't seem like `image-pixels` is being used?

**Review by riimuru — COMMENTED** [review]
Thank you for your pr!

File: bun.lockb
Comment (shimizudev) [reply bun.lockb:1]: Wait

**Comment by shimizudev** [comment]
FIxed @riimuru

**Comment by riimuru** [comment]
Thank you for the fix!

**Review by riimuru — APPROVED** [review]

**Comment by oddmario** [comment]
good job on that one, it has got a really complicated `sources` decryption logic 🫠

**Pull request merged**